agenda/presentation/src/main/java/com/gkp/agenda/presentation/home/AgendaUiState.kt

```diff
@@ -16,4 +20,26 @@ data class AgendaUiState(
 
     val uiDates: List<DateWithSelected>
         get() = getCurrentAndNextSixWeekDayDates(selectedDate)
+
+    val uiFormattedSelectedDate: String
+        @Composable
+        get() = when (selectedDate) {
+            LocalDate.now() -> {
+                stringResource(R.string.today)
+            }
```

Comment:
I like to use the following super helpful class to manage text and string resources. With it, I can use it within my data classes and ViewModel, and when inside my composables or with `Context`, I can then fetch the actual resource.

```
sealed interface UiText {
    data class DynamicString(val value: String): UiText
    class StringResource(
        @StringRes val id: Int,
        val args: Array<Any> = arrayOf()
    ): UiText

    @Composable
    fun asString(): String {
        return when(this) {
            is DynamicString -> value
            is StringResource -> stringResource(id = id, *args)
        }
    }

    fun asString(context: Context): String {
        return when(this) {
            is DynamicString -> value
            is StringResource -> context.getString(id, *args)
        }
    }
}
```